test/e2e-api/redis.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2022 The Numaproj Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+
+	"github.com/redis/go-redis/v9"
+)
+
+var redisClient *redis.Client
+
+func init() {
```

Comment:
More and more udsource/sink repos start using this to create redis. Should we move it to the utils repo?

Reply:
Yes that would be the better idea

Reply:
Can you create a task under the util repo to track this work?

Reply:
https://github.com/numaproj-contrib/numaflow-utils-go/issues/8